src/stepfunctions/steps/sagemaker.py

```diff
@@ -88,6 +95,12 @@ def __init__(self, state_id, estimator, job_name, data=None, hyperparameters=Non
 
             kwargs[Field.Resource.value] = get_service_integration_arn(SAGEMAKER_SERVICE_NAME,
                                                                        SageMakerApi.CreateTrainingJob)
+        # sagemaker.workflow.airflow.training_config does not accept ExecutionInput as input, only str, dict,
+        # TrainingInput or FileSystemInput. Transform placeholder into JSONpath to be processed and included in
+        # generated Sagemaker parameters. Placeholder for uri str only:
```

Comment:
I didn't really understand this comment.

Reply:
Agreed. This comment definitely need to be clearer and more concise! Will do in the next commit(s)!